### PR TITLE
rawagner_Select menu from toolbar of type HandledContributionItem

### PR DESCRIPTION
**Pull request by rhopp**

### CODE REVIEW

**Comment by rhopp** [comment]
test this

**Comment by rhopp** [comment]
test this

File: plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/lookup/MenuLookup.java [hunk 1]
```diff
@@ -124,7 +122,47 @@ public ActionContributionItem run() {
...
+								sendHide(item.getMenu(), true);
+								sendShowUI(item.getMenu());
+
+								// drill down
+								currentMenu = item.getMenu();
+								break;
+							} else {
+								
+								// we got a match
+								currentItem = item;
```
Comment (rawagner) [inline plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/lookup/MenuLookup.java]: how about adding a break here ? No need to continue in loop when we already have a match

Comment (rhopp) [inline plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/lookup/MenuLookup.java]: Thanks for noticing. Updated PR is on its way.

**Comment by rawagner** [comment]
retest this

**Comment by rhopp** [comment]
test this

**Comment by rawagner** [comment]
please check the failed test on windows machine, if it has something to do with this PR

**Pull request closed without merging**